requirements-base.txt

```diff
@@ -3,7 +3,7 @@
 beautifulsoup4>=4.7.1
 boto3>=1.22.12
 botocore>=1.25.12
-celery>=4.4.7
```

Comment:
Do we need to update any of the other requirements files or are those handled automatically?

* https://github.com/getsentry/sentry/blob/master/requirements-dev-frozen.txt#L21
* https://github.com/getsentry/sentry/blob/master/requirements-frozen.txt#LL19

Reply:
I ran `make freeze-requirements` but nothing came of it since `brotli` is already installed I guess.